test/test_parse.py

```diff
@@ -697,15 +739,16 @@ def test_numbers(self):
         self.assertEqual(normalize("that's eighteen nineteen twenty"),
                          "that is 18 19 20")
         self.assertEqual(normalize("that's one nineteen twenty two"),
-                         "that is 1 19 20 2")
+                         "that is 1 19 22")
```

Comment:
for these unittests we might want to throw a deprecation warning in case some downstream depends on this, relevant discussion https://github.com/MycroftAI/mycroft-core/issues/1962

Reply:
i dont think we need a warning, if anything dowtreamns is depending on this behavior it is already utterly broken for n > 20 in general... we would have received bug reports already.